lib/LANraragi/Plugin/Metadata/MEMS.pm

```diff
@@ -0,0 +1,161 @@
+package LANraragi::Plugin::Metadata::MEMS;
+
+use strict;
+use warnings;
+use Mojo::UserAgent;
+use Mojo::JSON qw(decode_json encode_json);
+use LANraragi::Utils::Logging qw(get_logger);
+
+# Meta-information about the plugin.
+sub plugin_info {
+  return (
+    # Standard metadata:
+    name => 'Mayriad\'s EH Master Script',
+    type => 'metadata',
+    namespace => 'memsplugin',
+    login_from => 'ehlogin',

```

Comment:
I was pretty happy to see another plugin depending on ehlogin at first, but it doesn't look like you actually need a useragent authentified to E-H at all since you're only consuming the API.

Reply:
Of course, if you rework the plugin to call the stock EH one, you'll need this so you can forward a `$lrr_info` object with the proper information

Reply:
Oh right, the login plugin is no longer needed. Back then I was going to add an option to check if a gallery has been removed and tag it accordingly, and that would require the login.